repo/content/builder.go

```diff
@@ -107,6 +109,15 @@ func (b packIndexBuilder) Build(output io.Writer) error {
 		return errors.Wrap(err, "error writing extra data")
 	}
 
+	randomSuffix := make([]byte, randomSuffixSize)
```

Comment:
This may not be required.

Reply:
Actually, for the sake of simplifying the reasoning, it makes sense to create a new index blob , even if that incurs having to upload a new (almost identical) index blob.
If this ends up being a common case, e.g., a mostly idle repo, then we can look into optimizing it.

Reply:
ack

Reply:
Jarek explained why this is needed.